en/docs/Learn/ConsumeAPI/Customizations/adding-a-user-signup-workflow.md

```diff
@@ -176,13 +176,13 @@ This section explains how to attach a custom workflow to the user signup operati
 
     Paths to the directory containing the client-truststore of each product are :
 
-        1. API-M - <API-M_HOME>/repository/resources/security
-        2. EI - <EI_HOME>/wso2/business-process/repository/resources/security
-        3. BPS - <BPS_HOME>/repository/resources/security
+        1. API-M - '<API-M_HOME>/repository/resources/security'
+        2. EI - '<EI_HOME>/wso2/business-process/repository/resources/security'
+        3. BPS - '<BPS_HOME>/repository/resources/security'
 
 #### Engaging the WS Workflow Executor in the API Manager
 
-1.  Log in to APIM management console ( `https://<Server Host>:9443/carbon` ) and select **Browse** under **Resources**.
+1.  Log in to APIM management console ( `https://<Server-Host>:9443/carbon` ) and select **Browse** under **Resources**.
```

Comment:
API-M is the abbreviation we use in paragraphs/sentences, except for URLS.

Reply:
Fixed